.github/actions/netlify-clean/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:lts-alpine
```

Comment:
Should we use a current version? I notice this produces some warnings:

```
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
29
```

Reply:
We only install `netlify-cli` (latest) and we don't have a way to force the installation of `core-js@>=3`. We could make a PR to `netlify-cli` to fix that but it might be a deeper dependency causing that.